google/cloud/storage/examples/storage_examples_common.cc

```diff
@@ -31,13 +33,15 @@ std::string MakeRandomBucketName(google::cloud::internal::DefaultPRNG& gen,
                                  std::string const& prefix) {
   // The total length of a bucket name must be <= 63 characters,
   static std::size_t const kMaxBucketNameLength = 63;
-  std::size_t const max_random_characters =
-      kMaxBucketNameLength - prefix.size();
+  auto const date =
+      absl::FormatCivilTime(absl::ToCivilDay(absl::Now(), absl::UTCTimeZone()));
```

Comment:
Nit: `#include "absl/time/time.h"` for `absl::ToCivilDay()`.

Reply:
Fixed.